GOVERNANCE.md

```diff
@@ -0,0 +1,158 @@
+# Prometheus Governance v1.0
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Team members**: Any members of the private [prometheus-team][team] Google group.
+
+* **Maintainers**: Team members who are leads on an individual project ([MAINTAINERS.md][maintainers.md]).
+
+* **Projects**: Any repository in the Prometheus [GitHub organization][gh].
+
+* **The Prometheus Project**: The sum of all activities performed under this governance, concerning one or more Repositories or the community.
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc]. Furthermore, the Prometheus community strives for kindness, giving feedback effectively and building a welcoming environment. The Prometheus developer community strives to operate by rough consensus for the majority of decisions and only resort to conflict resolution through a motion if consensus cannot be reached.
+
+## Projects
+
+Each project must have a MAINTAINERS.md file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the prometheus-users mailing list. Any new repositories should be first proposed on prometheus-team@ and following the voting procedures listed below. When a repository is no longer relevant, it should be moved to the prometheus-junkyard Github organization.
+
+## Decisionmaking
+
+### Team members 
+
+Team member status is given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
+
+New members may be proposed by any existing member by email to [prometheus-team][team] and voted on by [consensus](#consensus). If no consensus is reached, the new member is not accepted.
+
+Team members are added to the [GitHub organization][gh] and have commit rights to all projects. They should however respect the maintainers of each project.
+
+Team members may retire at any time by emailing [the team][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+Unless voted upon otherwise, membership changes take effect immediately. Unless voted upon otherwise, any membership change is announced to [the public developer mailing list][devs] as soon as it is in effect.
```

Comment:
Membership announcements are new, I added them. We do not have any public list of who actually is a project member (the github org comes close; we might be able to open the listing of the google group member list?).

I think this is a good idea but only if there is consensus on it.

Reply:
Should we also have a public list of who is currently on "team", and use that as the authoritative source? Like a markdown file somewhere? Because nobody external can check who is currently on the prometheus-team mailing list, and we haven't been transparent at all about that list. But I think team membership should not be secret. That also automatically rules out keeping removals secret though.

Otherwise I don't really care whether we announce additions/removals or not. Maybe slight pref towards not doing it == less noise.

Reply:
A list on the community page would make sense to me.

I'd have a preference towards avoiding noise.

Reply:
I'll remove the announcements and add the list.